### PR TITLE
functional and tested setTrieFlushIntervalClone plugin

### PR DESCRIPTION
**Pull request by philip-morlier**

### CODE REVIEW

File: core/plugin_hooks.go [hunk 1]
```diff
@@ -296,3 +297,24 @@ func pluginGetBlockTracer(hash common.Hash, statedb *state.StateDB) (*metaTracer
...
+
+func PluginSetTrieFlushIntervalClone(pl *plugins.PluginLoader, flushInterval time.Duration) time.Duration {
+	fnList := pl.Lookup("SetTrieFlushIntervalClone", func(item interface{}) bool{
+		_, ok := item.(func(time.Duration) time.Duration)
+		return ok
+	})
+	for _, fni := range fnList {
+		log.Error("len fn list", "len", len(fnList))
+		if fn, ok := fni.(func(time.Duration) time.Duration); ok {
+			return fn(flushInterval)
```
Comment (AusIV) [inline core/plugin_hooks.go]: ```flushInterval = fn(flushInterval)```

**Comment by AusIV** [comment]
See https://github.com/openrelayxyz/plugeth-utils/pull/18#discussion_r1112262027

Also, tests are failing, seemingly to do with a null pointer dereference in your hook.

File: core/plugin_hooks.go [hunk 2]
```diff
@@ -296,3 +298,27 @@ func pluginGetBlockTracer(hash common.Hash, statedb *state.StateDB) (*metaTracer
...
+		if fn, ok := fni.(func(time.Duration) time.Duration); ok {
+			flushInterval = fn(flushInterval) 
+		}
+	}
+	return flushInterval
+}
+
+func pluginSetTrieFlushIntervalClone(flushInterval time.Duration) time.Duration {
+	if plugins.DefaultPluginLoader == nil {
+		log.Warn("Attempting setTreiFlushIntervalClone, but default PluginLoader has not been initialized")
```
Comment (AusIV) [inline core/plugin_hooks.go:322]: This should have a `return` statement after it so we don't invoke the plugin with a nil DefaultPluginLoader
